src/App.css

```diff
@@ -291,135 +291,216 @@
-  transition: background-color 0.3s ease;
+  max-height: 100%;
+  display: flex;
+  flex-direction: column;
+  box-shadow: var(--ds-shadow-raised,0 1px 1px rgba(9,30,66,0.25));
+  transition: box-shadow 0.2s ease;
+}
+
+.column:hover {
+  box-shadow: var(--ds-shadow-raised,0 3px 6px rgba(9,30,66,0.15));
 }
 
 .column-header {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  margin-bottom: 16px;
-  padding-bottom: 8px;
+  padding: 14px 16px;
+  position: relative;
   border-bottom: 1px solid var(--border-color);
+  margin: 0 2px;
+}
+
+.column-header-left {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  flex: 1;
+  min-width: 0;
 }
 
 .column-header h2 {
   color: var(--text-primary);
-  font-size: 16px;
-  font-weight: 500;
+  font-size: 15px;
+  font-weight: 600;
   margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
 }
 
-.column-header-left {
-  display: flex;
-  align-items: center;
-  gap: 8px;
+.column-emoji {
+  font-size: 16px;
+  line-height: 1;
 }
 
 .column-header-right {
   display: flex;
   align-items: center;
-  gap: 8px;
-}
-
-/* Task Styles */
-.task-list {
-  min-height: 100px;
+  gap: 4px;
 }
 
-.task {
-  background: var(--task-bg);
-  border-radius: 4px;
-  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
-  padding: 12px 16px;
-  margin-bottom: 8px;
-  border: 1px solid var(--border-color);
-  color: var(--text-primary);
-  font-size: 14px;
-  transition: all 0.3s ease;
+.delete-column {
+  padding: 6px;
+  background: transparent;
+  border: none;
+  color: var(--text-secondary);
+  cursor: pointer;
+  border-radius: 3px;
   display: flex;
   align-items: center;
-  gap: 12px;
-  cursor: grab;
-  user-select: none;
+  justify-content: center;
+  font-size: 16px;
+  line-height: 1;
+  opacity: 0;
+  transition: all 0.1s ease;
 }
 
-.task:active {
-  cursor: grabbing;
+.column:hover .delete-column {
+  opacity: 1;
 }
 
-.task.dragging {
+.delete-column:hover {
   background: var(--hover-bg);
-  box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
-  opacity: 0.9;
-  transform: scale(1.02);
-  cursor: grabbing;
+  color: var(--text-primary);
+}
+
+/* Task List */
+.task-list {
+  padding: 8px;
+  overflow-y: auto;
+  flex: 1;
+  margin: 0 2px;
 }
 
 .task-list.dragging-over {
   background: var(--hover-bg);
-  border-radius: 4px;
+  margin: 0 6px;
   padding: 8px;
+  border-radius: 8px;
+}
+
+/* Task Card */
+.task {
+  background: var(--column-background);
+  border-radius: 8px;
+  box-shadow: var(--ds-shadow-raised,0 1px 1px rgba(9,30,66,0.25));
+  padding: 10px 12px;
+  margin-bottom: 8px;
+  border: none;
+  color: var(--text-primary);
+  font-size: 14px;
   transition: all 0.2s ease;
+  display: flex;
+  gap: 8px;
+  cursor: pointer;
+  position: relative;
+  min-height: 32px;
 }
 
 .task:hover {
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-  border-color: var(--primary-color);
   background: var(--hover-bg);
+  transform: translateY(-1px);
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
transform: translate3d(0, -1px, 0);
```
Multiple transform animations could benefit from hardware acceleration by using transform: translate3d() instead of translateY() to leverage GPU acceleration.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;